src/components/CheckboxInput/CheckboxInput.jsx

```diff
@@ -0,0 +1,12 @@
+import "./CheckboxInput.scss";
+
+const CheckboxInput = () => {
```

Comment:
Line 3: Please pass in props of functions to handle changes to the text input and checkbox input.